api/v1/clearScreen.js

```diff
@@ -0,0 +1,17 @@
+export const clearScreen = ({ path }) => {
```

Comment:
I'm not sure what this would do when running in node 😁... probably crash 😥...what would be the desired behavior?

Writes are cumulative right now so you can keep writing shapes to the display without clearing so writing an empty shape would do just that. My thinking was that you might want to be able to recompute and redraw just part of a model without touching the rest, but maybe that's not the right behavior